.github/workflows/e2e_tests.yml

```diff
@@ -7,19 +7,19 @@ jobs:
     name: Build Container and Run E2E Tests
     runs-on: ubuntu-18.04
     env:
-      WP_VERSION: 5.8.0
-      TRAVIS_MARIADB_VERSION: 10.5.5
+      WP_VERSION: 5.9.3
+      TRAVIS_MARIADB_VERSION: 10.8.3
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
       - uses: actions/setup-node@v2
         with:
           node-version: '10.16.0'
+      - run: git config --global url.https://github.com/.insteadOf git://github.com/
       - run: npm ci
-      - run: npm install jest --global
```

Comment:
May I also know why we need to remove jest?

Reply:
`jest` is part of local dependencies global install might be some workaround for GH actions that is no longer needed.
 Tests are performing without global `jest` package this means we can remove it from GH action pipe.

Reply:
> Global install might be some workaround for GH actions that is no longer needed.
> Tests are performing without global `jest` package this means we can remove it from GH action pipe.

I think it might be good if we confirm with a8c team about this? My worry is that, there is some underlying reason, they need to install `jest` in GH action, that we don't know.

Reply:
If that's the case, then I think I'm okay.